packages/init/pre-bundle-source.js

```diff
@@ -6,4 +6,4 @@
 
 // Use a package self-reference to go through the "exports" resolution
 // eslint-disable-next-line import/no-extraneous-dependencies
-import '@endo/init/pre.js';
+export * from '@endo/init/pre.js';
```

Comment:
Please consider `export { lockdown } from ...`. The hazard I have in mind is that a name might be added or removed from the originating module and cause strange name collisions in distant dependees.